addon/components/table.ts

```diff
@@ -11,9 +11,109 @@ import { htmlSafe } from '@ember/string';
 import { SafeString } from '@ember/template/-private/handlebars';
 import Media from 'ember-responsive';
 import NativeArray from '@ember/array/-private/native-array';
-import { ColumnValue, TableSort, RowClickEvent } from '@gavant/ember-table';
+// import { ColumnValue, TableSort, RowClickEvent } from '@gavant/ember-table';
```

Comment:
sine these were not externally accessible before, i think we should just remove the types from this file so we dont have them defined in 2 places, and just import them from this from if they are being used elsewhere within the addon. And then if we can figure out a better way to export them we can move them later.